File: app/src/main/java/com/android/collections/models/RegisterResponse.java
package com.android.collections.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RegisterResponse {

    @SerializedName("status")
    @Expose
    private Boolean status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("ID")
    @Expose
    private Integer iD;
    @SerializedName("fmctoken")
    @Expose
    private String fmctoken;
    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("user_mobile")
    @Expose
    private String userMobile;
    @SerializedName("email")
    @Expose
    private String email;

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Integer getID() {
        return iD;
    }

    public void setID(Integer iD) {
        this.iD = iD;
    }

    public String getFmctoken() {
        return fmctoken;
    }

    public void setFmctoken(String fmctoken) {
        this.fmctoken = fmctoken;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getUserMobile() {
        return userMobile;
    }

    public void setUserMobile(String userMobile) {
        this.userMobile = userMobile;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }


}

File: app/src/main/java/com/android/collections/models/RegisterResponse.kt
package com.android.collections.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RegisterResponse {

    @SerializedName("status")
    @Expose
    var status: Boolean? = null
    @SerializedName("message")
    @Expose
    var message: String? = null
    @SerializedName("ID")
    @Expose
    var id: Int? = null
    @SerializedName("fmctoken")
    @Expose
    var fmctoken: String? = null
    @SerializedName("username")
    @Expose
    var username: String? = null
    @SerializedName("user_mobile")
    @Expose
    var userMobile: String? = null
    @SerializedName("email")
    @Expose
    var email: String? = null


}
